be/src/exec/vectorized/aggregate/agg_hash_map.h

```diff
@@ -597,6 +606,8 @@ struct AggHashMapWithSerializedKey {
     std::unique_ptr<MemPool> mem_pool;
     uint8_t* buffer;
     ResultVector results;
+
+    int32_t _batch_size;
```

Comment:
```suggestion
    int32_t batch_size;
```   Only private variate name start with '_'.